src/Result.php

```diff
@@ -204,61 +181,50 @@ public function setEndMemory($endMemory)
 
     /**
      * Get the memory spike amount of the callback.
-     *
-     * @return float
      */
-    public function getMemory()
+    public function getMemory(): float
     {
         return $this->endMemory - $this->startMemory;
     }
 
     /**
      * Get the exception thrown by the callback.
-     *
-     * @return Exception|null
      */
-    public function getException()
+    public function getException(): ?\Throwable
     {
         return $this->exception;
     }
 
     /**
      * Set the exception thrown by the callback.
-     *
-     * @param Exception|null $exception
-     *
-     * @return $this
      */
-    public function setException($exception)
+    public function setException(?\Throwable $exception): self
```

Comment:
I changed these `Exception` to `Throwable`, since `Machine->executeMutedCallback()` actually catches that, thus it could end up here.